wiki/Module-Registration.md

```diff
@@ -1,10 +1,52 @@
-![Up to date as of 4.0.0](https://img.shields.io/static/v1?label=dnd5e&message=4.0.0&color=informational)
+![Up to date as of 4.3.0](https://img.shields.io/static/v1?label=dnd5e&message=4.3.0&color=informational)
 
 The system includes a number of features allowing modules and worlds to register information for the system directly in their manifests, rather than requiring separate scripts.
 
-## Compendium Types
+## Compendium Flags
 
-Compendiums can contain flags indicating what types of items or actors they contain. This is used to aid the Compendium Browser in quickly searching for content. More information about how this is set up can be found on the [Compendium Browser guide](Compendium-Browser.md#module-support).
+These flags should be applied to the `flags` object within an individual compendium definition inside the `packs` array.
+
+### Display Mode
+
+The `display` flag can be used to specify that a specific compendium should use the [Table of Contents][Table-of-Contents.md] display rather than the default compendium interface. It should only be used for compendiums with the `JournalEntry` document type.
+
+```json
+{
+  "name": "rules",
+  "label": "Rules (SRD)",
+  "system": "dnd5e",
+  "path": "packs/rules",
+  "type": "JournalEntry",
+  "private": false,
+  "flags": {
+    "display": "table-of-contents"
```

Comment:
We seem to support both `flags.display` and `flags.dnd5e.display`. Which one is preferred?

Reply:
Probably `dnd5e.display`, I should probably update the system manifest to use that rather than just `display`.